content/wasm-functions/using-cron-jobs.md

```diff
@@ -0,0 +1,237 @@
+
+    // Log every time the route is triggered, including the message (if any)
+    console.log(`Cron job triggered at ${now} with msg: "${msg}"`);
+
+    // Return a generic success response
+    return new Response("Cron job executed", {
+        status: 200,
+    });
+});
+
+// Attach the router to the fetch event
+addEventListener('fetch', (event) => {
+    event.respondWith(router.fetch(event.request));
+});
+```
+
+To build and run the application locally, run the following command:
+
+<!-- @selectiveCpy -->
+
+```console
+spin build --up
+```
+
+Let's curl the application's endpoint to test if it is working as expected:
+
+<!-- @selectiveCpy -->
+
+```console
+curl localhost:3000/cron-me
+```
+
+ You should see the following output in response:
+
+<!-- @nocpy -->
+
+```console
+Cron job executed
+```
+
+## Deploy Spin Application to Fermyon Wasm Functions
+
+Before scheduling our cron job, we must deploy our Spin application to the Fermyon Wasm Functions platform with the following command:
+
+<!-- @selectiveCpy -->
+
+```console
+spin aka deploy
+```
+
+Upon successful deployment, you should see output along the lines of:
+
+<!-- @nocpy -->
+
+```console
+App 'hello-cron-job' initialized successfully.
+Waiting for application to be ready... ready                                                                                   
+
+Application URL:   https://1064f8cc-4b83-4fb0-a550-ff51d8a85a18.aka.fermyon.tech/
+  Routes:
+  - hello-cron-job: https://1064f8cc-4b83-4fb0-a550-ff51d8a85a18.aka.fermyon.tech/cron-me
+```
+
+You can use the domain name provided to you by Fermyon Wasm Functions to test that the Spin application is working as expected by curling the new endpoint:
+
+<!-- @selectiveCpy -->
+
+```console
+curl https://1064f8cc-4b83-4fb0-a550-ff51d8a85a18.aka.fermyon.tech/cron-me
+```
+
+You should receive the same message as you did during local testing:
+
+<!-- @nocpy -->
+
+```console
+Cron job executed
+```
+
+# Scheduling a Cron Job for Your Spin Application
+
+Now we’ll use the `spin aka crons` command to invoke the Spin application's HTTP endpoint on a scheduled basis. The schedule uses standard crontab syntax.
+
+> `spin aka crons` supports multiple digits in each cron field, as well as intervals in any position (e.g., `* */12 * * *` to run every 12 hours). You can also use comma-separated lists and ranges. If your schedule includes a specific time of day, be sure to use UTC—you may need to [convert your local time to UTC](https://www.worldtimebuddy.com/?pl=1&lid=100&h=100&hf=1) to ensure correct execution.
+
+Let’s create a cron job that triggers your Spin application every 5 minutes. The `spin aka crons` create subcommand takes three arguments:
+
+* **Schedule** – How often the job runs, in standard cron syntax
+* **Path and query** – The HTTP path (and optional query parameters) to invoke
+* **Name** – A name for your cron job (this will be associated with the Spin application in your current working directory)
```

Comment:
Consider breaking this up - "it applies to the app in your current directory" is important enough that it shouldn't be a parenthesis under "the name for the job".  E.g.

```suggestion
* **Name** – A name for your cron job

The cron job will always invoke the Spin application in your current working directory.
```

(or perhaps "...will always invoke the Spin application deployed from your current..." which avoids any implication that it will invoke it _in_ your cwd, but is perhaps awkward and requires extra effort for the reader to parse, not sure)

Reply:
Good point about highlighting this. I removed it from parenthesis and put it at the beginning of the paragraph for this section.